dataactcore/migrations/versions/9199891101c5_added_external_data_load_date.py

```diff
@@ -0,0 +1,58 @@
+"""added external data load date
+
+Revision ID: 9199891101c5
+Revises: 321af67fae11
+Create Date: 2018-06-05 10:35:23.538191
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '9199891101c5'
+down_revision = '321af67fae11'
```

Comment:
A new migration was merged into dev branch. The migration will need to be updated to prevent conflicts.

Reply:
merging